sample/README.md

```diff
@@ -0,0 +1,130 @@
+# sample
```

Comment:
Do we need this autogen README at all?

- too verbose
- not adjusted to samlocal

=> remove?!